scheduler/scd_utils.py

```diff
@@ -0,0 +1,239 @@
+import datetime as dt
+import collections
+import shutil
+import locale
+
+# Standardizes datetime format to use the en_US formatting in the Python docs.
+locale.setlocale(locale.LC_TIME, "en_US")
```

Comment:
I've been thinking about this. It might be nice to stick with format YYYYMMDD just to standardize with the bfiq and rawacf filenames